Content.Shared/IdentityManagement/Components/IdentityBlockerComponent.cs

```diff
@@ -21,7 +21,8 @@ public enum IdentityBlockerCoverage
     NONE  = 0,
     MOUTH = 1 << 0,
     EYES  = 1 << 1,
-    FULL  = MOUTH | EYES
+    OUTER = 1 << 1,
```

Comment:
1 << 2, this is the same bit as EYES

Reply:
Didn't realize I did not adjust it. Fixed👍